inference-engine/src/transformations/include/ngraph_ops/convolution_ie.hpp

```diff
@@ -90,6 +114,7 @@ class TRANSFORMATIONS_API ConvolutionIE : public Op {
     CoordinateDiff m_pads_end;
     PadType m_auto_pad;
     size_t m_group;
+    element::Type m_output_type;
```

Comment:
Please add set methods for this attribute.

Reply:
The same comment for other ops.

Reply:
Do we really need setter for this member? I just didn't see use cases when it will be useful.

Reply:
It will be necessary for precision conversion pass.